app/templates/components/reminder-card.hbs

```diff
@@ -0,0 +1,14 @@
+<h2 class="spec-reminder-card--title">{{model.title}}</h2>
+<p class="spec-reminder-card--date">{{model.date}}</p>
+<p>{{model.notes}}</p>
+{{#link-to "reminders.reminder-card.edit"}}
+  <button class="spec-reminder-card--editbtn">edit</button>
+{{/link-to}}
+{{#link-to "reminders"}}
+  <button
+    class='spec-reminder--deletebtn'
+    {{action 'removeReminder' model on="click"}}>
```

Comment:
We don't need this `on='click'` bit here, buttons in Ember will assume you are targeting a click by default